src/structures/Manager.ts

```diff
@@ -228,8 +228,49 @@ export class Manager extends EventEmitter {
       });
   }
 
-  /**
-   * Initiates the Manager class.
+    /**
+     * Returns the Node in same region as server.
+     * @param region
+     */
+    public nearestNode(region: string): Node {
+        const nodes = this.nodes
+            .filter((node) => {
+                if (!node.options.region)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585535)
```suggestion
                if (!node.options.region)
```